### DISCUSSION TITLE
更改端口号问题

### DISCUSSION THREAD

**Discussion by xmoxmo** [whyour/qinglong#1669] [Q&A]
修改默认端口在扩展命令文件extra.sh中写入：

sed -i '/^  listen 5700;/c\  listen XXXXX;' /ql/docker/front.conf

1、如何能让这个命令更新面板后自动执行
2、如何才在在不重启容器的情况下让新端口生效

请不吝赐教，谢谢

**Comment by whyour** [comment]
改下映射的端口不行？为什么要改容器内到端口

**Reply by xmoxmo** [reply]
主要是我设置映射的时候，不显示访问的真实ip全是自身ip

**Reply by xmoxmo** [reply]
你的nginx转发的时候带上 proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for; 就行了

之前您给我这么回复过，我没有看明白这句放到哪里

如果容器重新创建，桥接改端口映射是不是需要重新创建容器

如果重新创建目录映射之前的目录原来的定时任务会完整保留吗(我改过定时任务的执行时间)？
